Formula/e/ente-cli.rb

```diff
@@ -0,0 +1,22 @@
+class EnteCli < Formula
+  desc "Utility for exporting data from Ente and decrypt the export from Ente Auth"
+  homepage "https://github.com/ente-io/ente/tree/main/cli"
+  url "https://github.com/ente-io/ente/archive/refs/tags/cli-v0.2.2.tar.gz"
+  sha256 "d03754880fe7dfdc422b37e4864c2aa469a2b19c645346de2d2de7fa62b71de3"
+  license "AGPL-3.0-only"
+  head "https://github.com/ente-io/ente.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cli" do
+      system "go", "build", *std_go_args(ldflags: "-s -w"), "main.go"
+      bin.install_symlink "ente-cli" => "ente"
+    end
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"ente"), "./cli"
```

Reply:
The command raises an error when passing `./cli` to it (it doesn't find the `go.mod` file). It's why I've used `cd cli` first.